src/commands/subscription.js

```diff
@@ -236,10 +259,14 @@ function processSubscriptionListCommand(bot, config, message) {
                 order: [
                     ['id', 'ASC']
                 ]
-			}).then(function(subscriptions) {
+            }).then(function(subscriptions) {
                 var response = "";
                 _.each(subscriptions, function(subscription) {
-                    response += "\nID {id} -> tell {target} when {event} for {source} in {league}".format(subscription.get());
+                    var context = subscription.get();
+                    if (_.startsWith(context['target'], 'channel_id:')) {
+                      context['target'] = 'your team channel';
```

Comment:
Right, it'd actually a group, so we'd need to add a groups property and a call to `groups.list`... May not be worth it for this little thing